src/github/code-scanning-alert.test.ts

```diff
@@ -52,10 +57,83 @@ describe("Code Scanning Alert Webhook Handler", () => {
 
 	const mockJiraRemoteLinkSubmit = jest.fn();
 	const mockJiraSecuritySubmit = jest.fn();
-	const jiraClient = {
+	const jiraClient: JiraClient = {
 		baseURL: JIRA_BASE_URL,
 		remoteLink: { submit: mockJiraRemoteLinkSubmit },
-		security: { submitVulnerabilities: mockJiraSecuritySubmit }
+		security: { submitVulnerabilities: mockJiraSecuritySubmit },
+		issues: {
```

Comment:
Wow, that's pretty cumbersome, but guess we don't have much options here, do we?

Reply:
Maybe to have some factory that would generate a deep mock and then just grab the functions/methods we are interested in? E.g.
```
const jiraClient: JiraClient = generateDeeplyMockedJiraClient();
mockJiraRemoteLinkSumit=jiraClient.remoteLink.submit;
```
Also not sure if it is suitable here though :)